### DISCUSSION TITLE
Examples of sustainability dashboards

### DISCUSSION THREAD

**Discussion by mxochicale** [SciKit-Surgery/sustainable-pkg-stats#32] [Ideas]
Opening this one to list examples of sustainability dashboards

**Comment by mxochicale** [comment]
Simulation Open Framework Arquitecture (SOFA) Dashboard. 
https://www.sofa-framework.org/dash/

**Comment by mxochicale** [comment]
Dashboard of supercomputer Fugaku
https://status.fugaku.r-ccs.riken.jp/d/fugaku/0-operation-status-of-fugaku?orgId=1

It is based on open-source project https://grafana.com/. Here you can see examples of dashboards https://grafana.com/grafana/dashboards/?pg=hp&plcmt=lt-box-dashboards

**Comment by thompson318** [comment]
http://www.astropy.org/astropy-dashboard/status.html

**Comment by thompson318** [comment]
There used to be a list of scikits, but it no longer works.
http://scikits.appspot.com/scikits

**Comment by yidilozdemir** [comment]
https://e4s-project.github.io/DocPortal.html

**Comment by yidilozdemir** [comment]
I really like the way they visualise the community activity here: https://ropensci.r-universe.dev/builds

Then the different tabs for the packages themselves and the contributors is nice

**Comment by mxochicale** [comment]
Found ossinsight "Deep insight into developers and repos on GitHub about stars, pull requests, issues, pushes, comments, reviews.." e.g,, https://ossinsight.io/analyze/renovatebot/renovate#overview

**Reply by thompson318** [reply]
and https://next.ossinsight.io/analyze/scikit-surgery#overview
